SprawdzanieNIPu/ConsoleApplication1/NIP.cs

```diff
@@ -1,72 +1,111 @@
+    /// </summary>
+    /// <param name="nr">Wprowadzony ciąg znaków</param>
+   public NIP(string nr)
+   {
+    this.numer = nr.Trim();
+   }
+   
+    /// <summary>
+    /// Gets a value indicating whether długość jest poprawna
+    /// </summary>
+   public bool DlugoscOK
+   {
+    get
+    {
+     return this.numer.Length == 10;
+    }
+   }
+   
+    /// <summary>
+    /// Gets - sumę kontrolną
+    /// </summary>
+   public int Kontrolna
+   {
+    get
+    {
+     int[] wagi = new int[] { 6, 5, 7, 2, 3, 4, 5, 6, 7 };
+     double x = 0;
+     double liczba = 0;
 
-        public int kontrolna
-        {
-            // zwraca sumę kontrolną
-            get
-            {
-                int[] wagi = new int[] {6, 5, 7, 2, 3, 4, 5, 6, 7};
-                double x=0;
-                double liczba=0;
+     for (int i = 0; i < wagi.Length; i++) 
+     {
+      x = Convert.ToDouble(this.numer.Substring(i, 1));
+                    liczba = liczba + (x * wagi[i]);
+     }
 
-                for (int i = 0; i < wagi.Length; i++) 
-                {
-                    x = Convert.ToDouble(numer.Substring(i, 1));
-                    liczba = liczba + x*wagi[i];
-                }
+     liczba = liczba % 11;
 
-                liczba = liczba % 11;
+     return (int)liczba;
+    }
+   }
 
-                return (int)liczba;
-            }
-        }
+    /// <summary>
+    /// Gets a value indicating whether numer NIP jest poprawny?
+    /// Idotyczne jest ścisłe stosowanie się do zasad StyleCopa, skoro piszę komentarzae po polsku
+    /// </summary>
+   public bool CzyPoprawna
+   {
+    get
+    {
+       return this.DlugoscOK && 
+        (this.Kontrolna == Convert.ToDouble(this.numer.Substring(9, 1)));
+    }
+   }
 
-        public Boolean czyPoprawna
+    /// <summary>
+    /// Gets a STRING indicating whether numer NIP jest poprawny?
+    /// </summary>
+    public string CzyPoprTekst
+    {
+        get
         {
-            get
-            {
-               return this.dlugoscOK && 
-                   (this.kontrolna == Convert.ToDouble(this.numer.Substring(9, 1)));
-            }
+                return this.CzyPoprawna ? "jest prawidłowy" : "jest niepoprawny";
         }
+    }
 
-        public string podziel
-        {
-            // zwraca ładnie sformatowany numer NIP
-            get
-            {
-                if (this.dlugoscOK)
-                {
-                    return string.Format("{0}-{1}-{2}-{3}",
-                        this.numer.Substring(0, 3),
-                        this.numer.Substring(3, 3),
-                        this.numer.Substring(6, 2),
-                        this.numer.Substring(8, 2));
-                }
-                else
-                {
-                    return this.numer;
-                };
-            }
-        }
+   /// <summary>
+   /// Gets - zwraca ładnie sformatowany numer NIP.
+   /// </summary>
+   public string Podziel
```

Comment:
Czemu private? To akurat wydaje się kandydat na public.

To może być overkill, ale warto poczytać jak można to ładnie zrobić:
https://msdn.microsoft.com/en-us/library/bb762932%28v=vs.110%29.aspx